src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -34,7 +34,8 @@ internal static class PredefinedCodeFixProviderNames
         public const string MoveToTopOfFile = "Move To Top Of File";
         public const string PopulateSwitch = "Populate Switch";
         public const string QualifyMemberAccess = nameof(QualifyMemberAccess);
-        public const string RemoveUnnecessaryCast = "Remove Unnecessary Casts";
+        public const string RemoveBraces = "Remove Unnecessary Casts";
```

Comment:
Uh, whoa.  That looks like a merge-with-conflicts artifact.  I hope I didn't get any others.